### DISCUSSION TITLE
Is MongoDB now supported ?

### DISCUSSION THREAD

**Discussion by valodes** [filamentphp/filament#13692] [Help]
### Package

Other

### Package Version

v3.2.96

### How can we help you?

# MongoDB Support for Laravel Filament

Hello Filament Community,

I hope everyone is doing well. I have a question regarding MongoDB support in Laravel Filament. Has MongoDB been integrated into the last version of Filament?

When I attempted to use MongoDB, I encountered the following error:

`This database engine does not support inserting while ignoring errors.`

I would appreciate any insights or updates on this matter. Thank you in advance for your help!

Best regards,
Valodes

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/discussions/6384#discussioncomment-5790368

**Comment by gannusravan** [comment]
Even i am facing the same issue with filament and mongodb
![image](https://github.com/user-attachments/assets/aeb78383-c71e-4293-84a5-9cf551c8767b)
if you have solved this problem please guide me to resolve this issue thanks in advance

**Reply by BukkitAT** [reply]
Configure mongodb as cache driver that fixes it for me
[See](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/v5.x/cache/)

**Reply by gmanikandan-dev** [reply]
It works fine. Thank you @CruxAllez
